### ISSUE TITLE
CountLimitedTextInput

### ISSUE THREAD

**Issue by ddfridley** [EnCiv/undebate-ssp#8]
```
<CountLimitedTextInput
   name="Question 2"
   maxCount=250
   defaultValue=""
   onDone=()=({valid: bool, value: any)
/>
```
Here's the figma: https://www.figma.com/file/IQKPx02pkBErpmhQoECoq9/Undebate?node-id=151%3A2458
![image](https://user-images.githubusercontent.com/3317487/128967810-fc6a1106-19e6-4ef7-b9e5-ccb58f132a39.png)

**Comment by ddfridley** [comment]
@SethBlakley I have updated the description of this to include an onDone call back.  See above.  Thanks.

**Comment by rkoleci** [comment]
Can you assign this to me ?

**Comment by ddfridley** [comment]
@rkoleci Welcome. I have assigned you.  If you fill out the [slack form](https://docs.google.com/forms/d/e/1FAIpQLSee58BUiy12dtloG9pLITsELcNldIwXcEtCotV9r95BZJSIVA/viewform?usp=sf_link) I can add you to the slack workspace.

**Comment by rkoleci** [comment]
I will work on this

**Comment by ddfridley** [comment]
Closed by #82
